lib/src/puppetdb-cli.cc

```diff
@@ -89,8 +90,9 @@ pdb_query(const json::JsonContainer& config,
         curl_free);
 
     const auto server_url = pdb_server_url(config) +
-            "/pdb/query/v4" +
-            "?query=" +
+            "/pdb/query/v4?" +
+            "pretty=" + (is_pretty ? "true" : "false") +
```

Comment:
FYI I don't think we'll want to use this as it prevents us from using POST, because `pretty=` does not work with POST in PDB.